deploy/gk-deploy

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+. ./gk-functions
```

Comment:
why can't we do source gk-functions? I mean I just came to know about source when I tried the unit testing part. I just wanna know is there any advantage in this/source.